src/app.js

```diff
@@ -38,8 +37,8 @@ export default function (app) {
     }
 
     if (obj = plugin.hooks) {
-      Object.keys(obj).forEach(function (key) {
-        hooks[key].push(obj[key])
+      Object.keys(obj).map(function (key) {
```

Comment:
this obsessive bytes-first mindset is gonna bite you.

https://jsperf.com/object-keys-vs-for-in-with-closure/114
https://jsperf.com/object-keys-vs-for-in-with-closure/137